rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -122,18 +123,20 @@ separator nonempty Case "" ;
 -- Name Declarations.
 -- Eventually will have IOPairs.
 NameDeclSimpl. NameDecl ::= Var ;
-NameDeclUrn. NameDecl ::= Var "(" Uri ")" ;
+NameDeclUrn. NameDecl ::= Var "(" UriLiteral ")" ;
 separator nonempty NameDecl "," ;
 
 -- Booleans:
-BoolTrue.   Bool ::= "true" ;
-BoolFalse.  Bool ::= "false" ;
+BoolTrue.   BoolLiteral ::= "true" ;
+BoolFalse.  BoolLiteral ::= "false" ;
 -- Ground types:
-GroundBool.    Ground ::= Bool ;
-GroundInt.     Ground ::= Long ;
-GroundString.  Ground ::= String ;
-GroundUri.     Ground ::= Uri ;
-token Uri ('`' ((char - ["\\`"]) | ('\\' ["`\\"]))* '`') ;
+-- The "Literal" suffix avoids collisions with Simple Types
+GroundBool.    Ground ::= BoolLiteral ;
+GroundInt.     Ground ::= LongLiteral ;
+GroundString.  Ground ::= StringLiteral ;
+GroundUri.     Ground ::= UriLiteral ;
+token StringLiteral ( '"' ((char - ["\"\\"]) | ('\\' ["\"\\nt"]))* '"' );
```

Comment:
`StringLiteral` is not the same as `UriLiteral`.

Reply:
They are not defined the same. Should they?

Reply:
My bad, I saw backtick but it was a quote. :+1: